jobs/ShareMethod.groovy

```diff
@@ -200,4 +200,47 @@ def downloadManifest(String url, String target){
     }
 }
 
+def generateManifestFromPR(String manifest_path){
+    def manifest = new net.sf.json.JSONObject()
+    String repository_url = "https://github.com/${ghprbGhRepository}.git"
```

Comment:
Because is already defined by Jenkins plugin "ghprb".
We can access it directly.
Sometimes, ```withEnv``` just used to tells users the script need to pre-defined some jenkins parameters or environment variables.